application/models/gateway/Panacea_model.php

```diff
@@ -15,17 +15,17 @@
 /**
  * panacea_model Class
  *
- * Handle all messages database activity 
+ * Handle all messages database activity
  * for Panacea Mobile <http://panaceamobile.com>
  *
  * @package		Kalkun
  * @subpackage	Messages
  * @category	Models
  */
-require_once('nongammu_model'.EXT);
+require_once('nongammu_model.php');
```

Comment:
Wouldn't this also need to be case sensitive?

Reply:
The PHP files in the repo all finish lower case, don't they? So not needed I think.

Reply:
Previously, they were, but now they are being changed into camel case with your PR

```
application/models/gateway/Nongammu_model.php
```

Reply:
Oh, I thought you talked about php extension. You're right, this probably has to be changed. Will do.

Reply:
Commit updated